client/ayon_houdini/plugins/publish/validate_render_product_paths_unique.py

```diff
@@ -0,0 +1,147 @@
+import inspect
+from collections import defaultdict
+from typing import Dict, List
+
+import pyblish.api
+import clique
+import hou
+
+from ayon_core.pipeline import (
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+
+from ayon_houdini.api import plugin
+from ayon_houdini.api.action import SelectInvalidAction
+
+
+def get_instance_expected_files(instance: pyblish.api.Instance) -> List[str]:
+    """Get the expected source render files for the instance."""
+    # Prefer 'expectedFiles' over 'frames' because it usually contains more
+    # output files than just a single file or single sequence of files.
+    expected_files: List[Dict[str, List[str]]] = (
+        instance.data.get("expectedFiles", [])
+    )
+    filepaths: List[str] = []
+    if expected_files:
+        # Products with expected files
+        # This can be Render products or submitted cache to farm.
+        for expected in expected_files:
+            # expected.values() is a list of lists
+            filepaths.extend(expected.values())
```

Comment:
This one should be (otherwise this error will occur `unhashable type 'list'` on line 111)
```suggestion
            for output_files in expected.values():
                filepaths.extend(output_files)
```